tests/unit/phpDocumentor/Transformer/Router/UrlGenerator/Standard/FqsenDescriptorTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * phpDocumentor
+ *
+ * PHP Version 5.3
+ *
+ * @copyright 2010-2014 Mike van Riel / Naenius (http://www.naenius.com)
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Transformer\Router\UrlGenerator\Standard;
+
+use Mockery as m;
+use phpDocumentor\Reflection\DocBlock\Tags\Reference\Fqsen;
+use phpDocumentor\Reflection\Fqsen as RealFqsen;
+
+/**
+ * Test for the MethodDescriptor URL Generator with the Standard Router
+ */
+class FqsenDescriptorTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @covers phpDocumentor\Transformer\Router\UrlGenerator\Standard\FqsenDescriptor::__invoke
+     * @covers phpDocumentor\Transformer\Router\UrlGenerator\Standard\QualifiedNameToUrlConverter::fromClass
+     * @dataProvider provideFqsens
+     */
+    public function testGenerateUrlForFqsenDescriptor($fromFqsen, $toPath)
+    {
+        // Arrange
+        $realFqsen = new RealFqsen($fromFqsen);
+        $fqsen = new Fqsen($realFqsen);
+        $fixture = new FqsenDescriptor();
+
+        // Act
+        $result = $fixture($fqsen);
+
+        // Assert
+        $this->assertSame($toPath, $result);
+    }
+
+    /**
+     * @covers phpDocumentor\Transformer\Router\UrlGenerator\Standard\FqsenDescriptor::__invoke
+     */
+    public function testFqsenDescriptorReturnsFalseWhenNodeOfWrongType()
+    {
+        // Arrange
+        $fqsen = m::mock('phpDocumentor\Reflection\DocBlock\Tags\Reference\Reference');
+        $fixture = new FqsenDescriptor();
+
+        // Act
+        $result = $fixture($fqsen);
+
+        // Assert
+        $this->assertSame(false, $result);
+    }
+
+    public function provideFqsens()
+    {
+        return array(
+            array('\\My\\Space\\Class', '/classes/My.Space.Class.html'),
```

Comment:
Looks like you are missing a namespace conversion here. Not sure how to detect the difference between a namespace and class because the fqsen are equal. And can even overlap. So I think this is a short comming to the full implementation stack. 
In other parts of phpdocumentor namespaces are prefixed with a `~`. But that won't work here since that's not an allowed FQSEN format. e.g `~\\My\\Space` 

Think we should document that somewhere?